src/dbnode/storage/repair_test.go

```diff
@@ -66,7 +66,7 @@ func TestDatabaseRepairerStartStop(t *testing.T) {
 		return nil
 	}
 
-	repairer.Start()
+	go repairer.Run()
```

Comment:
nit: I think the `.Start()` semantics are a little clearer? As far as I can tell, nothing outside of test code uses `.Run()` without wrapping it in a goroutine; also makes it easier to reason about when building the background task itself, as it will be able to dictate whether or not it runs in a goroutine rather than relying on the callers using this correctly

Reply:
I had mixed feelings about this. I made the change mainly to make this code in `mediator.go` look consistent for launching static/dynamic processes and making it obvious that each of them runs on a separate goroutine:
```go
        go m.ongoingFileSystemProcesses()
	go m.ongoingColdFlushProcesses()
	go m.ongoingTick()

	for _, process := range m.backgroundProcesses {
		go process.Run()
        }
```
LMK if this piece changes your opinion or not.